spec.md

```diff
@@ -136,6 +128,12 @@ together.
 
 ## Box
 
+The Tox protocol differentiates between two types of text: Plain Text and
+Cipher Text. Cipher Text may be transmitted over untrusted data channels. Plain
+Text can be Sensitive or Non Sensitive. Sensitive Plain Text must be
+transformed into Cipher Text using the encryption function before it can be
+transmitted over untrusted data channels.
```

Comment:
LGTM

I've made previous comments on the language here, and I stand by them. But that's no reason to stall this.